### PR TITLE
NXP-29338: add slack configuration

### PR DESCRIPTION
**Pull request by NourNuxeo**
Added slack plugin configuration for jenkins

### CODE REVIEW

File: config/credentials.yaml [hunk 1]
```diff
@@ -17,3 +17,9 @@ credentials:
         domain:
           description: "GitHub API domain"
           name: "api.github.com"
+      - credentials:
+          - string:
+              scope: GLOBAL
+              id: slack-token
+              secret: '${SLACK_TOKEN}'
+              description: Slack token
```
Comment (ataillefer) [inline config/credentials.yaml]: Could sort alphabetically and be consistent about single/double/no quotes regarding the other credentials.
If we choose to never use them, why not, then please add a first commit to clean up the file.

Comment (kevinleturc) [inline config/credentials.yaml]: Can you align the order of fields and the double quote on the format of the existing credentials?

File: config/unclassified.yaml [hunk 2]
```diff
@@ -15,3 +15,6 @@ unclassified:
         url: "https://jira.nuxeo.com/"
   timestamper:
     allPipelines: true
+  slackNotifier:
```
Comment (ataillefer) [inline config/unclassified.yaml:16]: should be placed before `timestamper`

File: config/unclassified.yaml [hunk 3]
```diff
@@ -15,3 +15,6 @@ unclassified:
...
+    teamDomain: "nuxeo"
+    tokenCredentialId: slack-token
```
Comment (ataillefer) [inline config/unclassified.yaml]: consistency about quotes

File: plugins.txt [hunk 4]
```diff
@@ -28,6 +28,7 @@ pipeline-model-extensions:1.7.1
 pipeline-utility-steps:2.6.1
 plain-credentials:1.7
 rebuild:1.31
+slack:2.41
```
Comment (ataillefer) [inline plugins.txt]: can you please sort alphabetically

**Review by ataillefer — DISMISSED** [review]

**Review by ataillefer — APPROVED** [review]

**Review by kevinleturc — APPROVED** [review]

**Pull request merged**